pycqed/analysis_v2/timedomain_analysis.py

```diff
@@ -334,14 +336,16 @@ def process_data(self):
         if self.cal_states_rotations is not None:
             self.cal_states_analysis()
         else:
+            # this assumes data obetained with classifier detector!
+            # ie pg, pe, pf are expected to be in the value_names
```

Comment:
Could we check for this and add an error if it is not the case ? or is there no way of doing that ? Because I think I ran in this issue more than once and it is not that easy to trace it back (you get a key error somewhere downstream).

Reply:
I don't know if I would spend time on improving analysis_v2; this should encourage people to switch to v3 and help with development!